lib/bolt/outputter/human.rb

```diff
@@ -311,6 +311,11 @@ def print_module_list(module_list)
         end
       end
 
+      def print_targets(options)
+        targets = options[:targets].map(&:name)
+        puts targets.join("\n"), "\nNUMBER OF TARGETS: #{targets.count}"
```

Comment:
bikeshed: How do we feel about this capitalization?

I think it makes sense to do something that makes this stand apart from target names but just having spaces should do that and we have color to utilize as well?

Reply:
Ya, I'm not fully happy with the capitalization. Latest commit switches to just the number in green text, which looks a bit nicer.

Reply:
I think "#{targets.count} targets" in green is better. Just the number is a bit cryptic.